container-registry/vulnerability-notification-function/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Is there a reason you are putting a seperate license here?  It isn't nescessary or appropiate.

Reply:
copied over from the original repo. removed.